tools/kagami/src/validator.rs

```diff
@@ -22,13 +22,11 @@ pub fn compute_validator_path() -> Result<PathBuf> {
     // command, otherwise it is called as a standalone
     std::env::var("CARGO_MANIFEST_DIR").map_or_else(
         |_| {
-            let source_dir = tempfile::tempdir()
-                .wrap_err("Failed to create temp dir for default runtime validator sources")?;
-            let out_dir = AbsolutePath::try_from(source_dir.path().join(DIR_CLONE))?;
+            let out_dir = AbsolutePath::try_from(PathBuf::from(".").join(DIR_CLONE))?;
             swarm::shallow_git_clone(GIT_ORIGIN, GIT_REVISION, &out_dir)?;
```

Comment:
So, now the repo is cloned directly to the working dir. Why is it a more preferable default than using a temporary directory?

Reply:
It seems to work. But I don't honestly think that it is preferable.